pkg/crc/config/validations.go

```diff
@@ -13,10 +13,11 @@ import (
 // ValidateBool is a fail safe in the case user
 // makes a typo for boolean config values
 func ValidateBool(value interface{}) (bool, string) {
-	if cast.ToString(value) == "true" || cast.ToString(value) == "false" {
-		return true, ""
+	if _, err := cast.ToBoolE(value); err != nil {
+		return false, "must be 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False"
```

Comment:
This is not very user friendly, I would only list one valid pair (`true/false`?). The fact that more values can be used is nice, can be added to our documentation if we think it's important, but here it's too much information to have everything.

Reply:
the other options is an additional... and is more to be contextually correct, ike saying "yes" or "no".